contrib/pg_stat_statements/sql/pg_stat_statements.sql

```diff
@@ -1,5 +1,9 @@
 CREATE EXTENSION pg_stat_statements;
 
+-- Known issue: query is not added to pg_stat_statements statistics in
+-- case it is planned by GPORCA. So disable GPORCA during tests.
+SET optimizer=off;
```

Comment:
shouldn't we move it to enable_pg_stat_statements? And I guess we need to reset the GUCin the end, if we set it.

Reply:
Moved to enable_pg_stat_statements, added reset in disable_pg_stat_statements

Reply:
Doesn't work for some reason on CI, even though it works locally. Maybe a better solution would be to only run the tests if optimizer is disabled, instead of disabling it in runtime?

Reply:
In this case I'm Ok to get this change back.